data/jobs.txt

```diff
@@ -1477,6 +1477,74 @@ mission "Core Pirate Attack"
 	on complete
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
+		
+mission "Raider Attack 1"
+	name "Defend <planet>"
+	description "Defeat an alien raid on <destination>."
+	minor
+	repeat
+	to offer
+		"combat rating" > 550
+		random < 7
+	source
+		planet Sheratan
+		planet Shangri-la
+		planet Nimbus
+	npc evade
+		personality timid unconstrained uninterested
+		government Korath
+		fleet "Korath Raid" 2
+	npc
+		personality heroic staying vindicitive
```

Comment:
Incorrect spelling: `vindicitive` -> `vindictive`

(As personalities are all reserved keywords (the modder can't just drop in a random word here and have it do anything) they don't get quoted.)